src/storage/QueryBaseProcessor.inl

```diff
@@ -34,20 +34,40 @@ bool QueryBaseProcessor<REQ, RESP>::validOperation(nebula::cpp2::SupportedType v
     return true;
 }
 
+template <typename REQ, typename RESP>
+void QueryBaseProcessor<REQ, RESP>::addDefaultProps(std::vector<PropContext>& p, EdgeType eType) {
+    p.emplace_back("_src", eType, 0, PropContext::PropInKeyType::SRC);
+    p.emplace_back("_rank", eType, 1, PropContext::PropInKeyType::RANK);
+    p.emplace_back("_dst", eType, 2, PropContext::PropInKeyType::DST);
+}
+
+template <typename REQ, typename RESP>
+void QueryBaseProcessor<REQ, RESP>::initContext(const std::vector<EdgeType>& eTypes,
+                                                bool need_default_props) {
+    std::transform(eTypes.cbegin(), eTypes.cend(),
+                   std::inserter(edgeContexts_, edgeContexts_.end()),
+                   [this, need_default_props](const auto& ec) {
```

Comment:
ec is edge type?

Reply:
ec is edgeContext

Reply:
I see that pass ec to addDefaultProps directly, and the second parameter is "EdgeType eType", so I think the ec is edge type.